### PR TITLE
Removed outdated legacy function to pass PHPUnit tests with MOODLE_402_STABLE

### PR DESCRIPTION
**Pull request by opitz**

### CODE REVIEW

**Comment by aspark21** [comment]
@mchurchward Is there anything you'd like us to do with this to get these fixes properly upstreamed?

File: version.php
Comment (mchurchward) [inline version.php:1]: Please revert these changes. We will update the version file when we release.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -36,7 +33,7 @@ jobs:
           - 5432:5432
 
       mariadb:
-        image: mariadb:10.5
+        image: mariadb:10
```
Comment (mchurchward) [inline .github/workflows/ci.yml:36]: I don't believe this change was necessary.

**Review by mchurchward — CHANGES_REQUESTED** [review]
Please respond to my review comments.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (aspark21) [reply .github/workflows/ci.yml:36]: https://github.com/moodlehq/moodle-plugin-ci/blob/master/gha.dist.yml#L20

the 10 tag points to the latest version in the 10.x series so 10.11.y
Whereas 10.5 limits to 10.5.y
https://hub.docker.com/_/mariadb/tags?page=2

it was pinned at some point but issue  was resolved so pin aught to be removed

Comment (aspark21) [reply .github/workflows/ci.yml:36]: https://github.com/moodlehq/moodle-plugin-ci/commit/4907b25c05d82bc616f3f907cc76d9b9339d3017

Comment (mchurchward) [reply .github/workflows/ci.yml:36]: Okay. But in the spirit of once issue per change, this does not need to be here. If we need to update the ci file, we will need another issue/pull request.

**Review by mikechurchward — CHANGES_REQUESTED** [review]
Just to be clear, once you have removed the changes to ci.yml and version.php, I will accept and merge this pull request.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (aspark21) [reply .github/workflows/ci.yml:36]: @opitz

**Comment by opitz** [comment]
Closing this as replaced with #74 and #75.
